### PR TITLE
LPS-93078 Add "My Sites" menu item to Personal Menu for quick inter-site navigation

### PR DESCRIPTION
**Pull request by stsquared99**

### CODE REVIEW

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by stsquared99** [comment]
ci:test:relevant

File: modules/apps/product-navigation/product-navigation-personal-menu-api/src/main/java/com/liferay/product/navigation/personal/menu/PersonalMenuEntry.java [hunk 1]
```diff
@@ -64,12 +66,17 @@ public String getPortletURL(HttpServletRequest request)
...
+	 *
+	 * @param  portletRequest the portlet request
 	 * @param  permissionChecker the permission checker
 	 * @return <code>true</code> if the entry should be displayed in the user
 	 *         personal menu; <code>false</code> otherwise
 	 * @review
 	 */
-	public default boolean isShow(PermissionChecker permissionChecker) {
+	public default boolean isShow(
+			PortletRequest portletRequest, PermissionChecker permissionChecker)
```
Comment (pei-jung) [inline modules/apps/product-navigation/product-navigation-personal-menu-api/src/main/java/com/liferay/product/navigation/personal/menu/PersonalMenuEntry.java]: Is there any reason we need to use `PortletRequest` here not `HttpServletRequest`?

**Comment by pei-jung** [comment]
ci:test

**Comment by pei-jung** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/70858. Thanks!

**Pull request closed without merging**